apps/ui/src/components/ProposalResults.vue

```diff
@@ -93,12 +93,12 @@ const otherResultsSummary = computed(() => {
     <div>
       <a
         href="https://blog.shutter.network/announcing-shutter-governance-shielded-voting-for-daos/"
+        class="text-skin-text"
```

Comment:
We could add `flex items-center` here, and remove the `inline-block` on both child elements